cves/kernel/CVE-2016-8650.yml

```diff
@@ -129,10 +132,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: 
```

Comment:
Fill this with information whether a unit test found this issue (and for fix_answer: is a unit test added to ensure the bug doesn't surface again)